gatsby-config.js

```diff
@@ -73,12 +73,6 @@ module.exports = {
         ],
       },
     },
-    {
```

Comment:
This looks like we're removing GA from the whole site?

Reply:
@ksigler7 We're just removing the GA Gatsby plugin. We're keeping the Google Tag Manager plugin, which loads GA via Google Tag Manager.

Reply:
cool - thanks for the explanation